src/responsive-layout.tsx

```diff
@@ -73,4 +77,29 @@ const ResponsiveLayout: React.FC<Props> = ({children, ...props}) => (
     </InternalResponsiveLayout>
 );
 
+export const ResetResponsiveLayout: React.FC<{
+    children: React.ReactNode;
+    mobile?: boolean;
+    desktop?: boolean;
```

Comment:
it is strange to have boolean props with `true` as default value. What about having `skipMobile` and `skipDesktop` props?

so, instead of `desktop={false}`, just `skipDesktop`

Reply:
makes sense, updated